src/helpers/avatar.ts

```diff
@@ -0,0 +1,9 @@
+async function downloadAvatar(assetsUrl: string, username: string): Promise<void> {
+    const response = await fetch(assetsUrl);
```

Comment:
Avatars are loaded from the assets service. See `load_balancer.conf` for configuration details